docs/database.md

```diff
@@ -0,0 +1,51 @@
+# Connect to database
+
+## local
+
+1. `psql -h localhost -p 5678 -U prisma -W`
+2. enter password `prisma`
+3. run `set search_path to "default$default";`
+
+## production
+
+1. in a terminal window run
+`cloud_sql_proxy -instances=moocfi:europe-north1:quizzes-backend=tcp:0.0.0.0:3306`
+to proxy production to localhost port 3306
+
+when connecting first time, run the following to get db_name and db_secret
+
+1. `kubectl get secrets`
```

Comment:
`kubectl` needs to be separately installed; also needs credentials set to access the cluster: https://cloud.google.com/kubernetes-engine/docs/how-to/cluster-access-for-kubectl - `gcloud container clusters get-credentials <CLUSTER_NAME>`

Change to correct namespace using `kubectl ns moocfi`

Reply:
Added :+1: